tensorflow/python/ops/math_ops.py

```diff
@@ -1163,7 +1163,7 @@ def multiply_no_nan(x, y, name=None):
   For example:
   >>> x = 0.0
   >>> y = 34.0
-  >>> tf.math.multiply_no_nan(0.0, 34.0)
+  >>> tf.math.multiply_no_nan(x, y)
```

Comment:
I think you should put a NaN as input so people understand that 0 * NaN = 0 for mul_no_nan

Reply:
@alextp I added the necessary changes, can you please take a look at the changes that I had made?